src/datasource/datasource.js

```diff
@@ -223,11 +226,11 @@ export class DataSource {
 
     async getBalance() {
         const pocket = await this.getPocketInstance()
-        const pocketAddress = config.address
+        const pocketAddress = "F7E8637E13033279095CFCC9E50548EC78E4C0FD";//config.address
```

Comment:
This shouldn't be a hardcoded address should it?

Reply:
nope, thanks